src/pages/s.tsx

```diff
@@ -56,7 +56,7 @@ function Page(props: Props) {
         Sections: Components imported from '../components/sections' only.
         Do not import or render components from any other folder in here.
       */}
-      <h1 className="temp-offscreen">{title}</h1>
+      <h1>{title}</h1>
```

Comment:
This is causing the following on PDPs:
![CleanShot 2022-01-12 at 20 05 13](https://user-images.githubusercontent.com/381395/149237315-f5ca8327-238d-40dc-ba92-2f5ffe05e414.png)